conf/logback.xml

```diff
@@ -53,6 +53,8 @@
         <appender-ref ref="STDOUT"/>
     </logger>
 
+    <logger name="org.slf4j" level="OFF" />
+
```

Comment:
Was it this validator that printed Json at Debug level? We should explicity exclude that logger even through debug isn't on by default on prod.